recipes/r-base/meta.yaml

```diff
@@ -0,0 +1,125 @@
+# Ported from Continuum Analytics: https://github.com/conda/conda-recipes/tree/master/r-base
+
+{% set name = "r-base" %}
+{% set native = 'm2w64-' if win else '' %}
```

Comment:
I have mixed feelings about this. It is a nice hack, but there is no guarantee there will always be a correspondence of packages. What do you think @mingwandroid? Maybe spelling out all the packages and add pre-processor selector is better...